Apps/WebClient/src/ClientApp/app/store/modules/auth/mutations.ts

```diff
@@ -2,6 +2,7 @@ import Vue from "vue";
 import { MutationTree } from "vuex";
 import { AuthState } from "@/models/storeState";
 import { User as OidcUser } from "oidc-client";
+import axios from "axios";
```

Comment:
I dont think the mutations should be using axios since it should not be doing anything else than setting the auth store